verified-answers.yaml

```diff
@@ -14,3 +16,1551 @@
       title: MongoDB Atlas Registration
     - url: https://www.mongodb.com/basics/mongodb-atlas-tutorial?tck=mongodb_ai_chatbot
       title: MongoDB Atlas Tutorial
+- answer: >-
```

Comment:
very nitty but I kinda wish the questions came before the answer

Reply:
Yah same ngl